src/josepy/jwk.py

```diff
@@ -16,15 +18,15 @@
 logger = logging.getLogger(__name__)
 
 
-class JWK(json_util.TypedJSONObjectWithFields):
+class JWK(json_util.TypedJSONObjectWithFields, metaclass=abc.ABCMeta):
     # pylint: disable=too-few-public-methods
     """JSON Web Key."""
     type_field_name = 'kty'
-    TYPES = {}  # type: dict
-    cryptography_key_types = ()  # type: tuple
+    TYPES = {}  # type: Dict
```

Comment:
Is it this?

```suggestion
    TYPES: Dict[str, Type] = {}
```

Reply:
I read through the documentation of Typing, and I don't see `Type` in the official documentation. It makes me a bit uncomfortable going with this suggestion (I committed the change, but it can always be handled locally). I'll search for an alternative that's documented.